Environment_Engine/Create/Gains.cs

```diff
@@ -117,13 +117,30 @@ public static People People(double sensible = 0.0, double latent = 0.0, Profile
                 Name = name,
             };
         }
-
+        [Description("Create a Plug Gain object which can be attributed to a space")]
```

Comment:
Please add a blank line here before the description tag to separate the previous method from this one